docs/services/slam/cartographer/_index.md

```diff
@@ -62,42 +62,24 @@ Before adding a SLAM service, you must follow [these instructions](/program/exte
 ### Add a SLAM Service
 
 {{< tabs name="Add the Cartographer Service Live">}}
-{{% tab name="Config Builder" %}}
-
-Go to your robot's page on the [Viam app](https://app.viam.com/).
-Navigate to the **config** tab on your robot's page, and click on the **Services** subtab.
-
-Add a service with type `slam`, model `cartographer`, and a name of your choice:
-
-![adding cartographer slam service](../img/run_slam/add-cartographer-service-ui.png)
-
-Paste the following into the **Attributes** field of your new service:
-
-```json
-{
```

Comment:
Is this no longer needed?

Reply:
We can't use the Builder because as far as I can see, there is no way to pick a model that's not `rdk:builtin:MY_MODEL`. 

Pre-populated from Raw JSON you can see this:

<img width="734" alt="Screenshot 2023-04-21 at 10 35 08 AM" src="https://user-images.githubusercontent.com/3912676/233663246-9d4addb3-b083-442e-a167-09dfcc4ae4a8.png">

But choosing it in the builder is not possible:

<img width="744" alt="Screenshot 2023-04-21 at 10 35 52 AM" src="https://user-images.githubusercontent.com/3912676/233663420-c36567be-a16c-4f73-ba88-30e7049fa6ea.png">

Reply:
PS: The Rplidar module documentation does not have a tab for the builder, either, probably for the same reason.